pjsip/src/pjsip-ua/sip_inv.c

```diff
@@ -5154,8 +5154,10 @@ static void inv_on_state_confirmed( pjsip_inv_session *inv, pjsip_event *e)
 		    status = pjmedia_sdp_neg_send_local_offer(inv->pool_prov,
 							      inv->neg, 
 							      &active_sdp);
-		    if (status == PJ_SUCCESS)
+		    if (status == PJ_SUCCESS) {
 			sdp = (pjmedia_sdp_session*) active_sdp;
+			sdp->origin.version++;
+		    }
```

Comment:
Shouldn't it be done in `pjmedia_sdp_neg_send_local_offer()`? Most SDP version increases are handled there. Also it should compare active SDP vs initial SDP first if `PJMEDIA_SDP_NEG_COMPARE_BEFORE_INC_VERSION` is set, similar to `pjmedia_sdp_neg_modify_local_offer2()` that compares new offer vs initial SDP.